applications/nrf5340_audio/src/audio/audio_system.h

```diff
@@ -22,6 +22,16 @@
  */
 int audio_encode_test_tone_set(uint32_t freq);
 
+/**
+ * @brief Step through different test tones
+ *
+ * @note A stream must already be running to use this feature.
+ * Will step through test tones: 1 kHz, 2 kHz, 4 kHz and off.
+ *
+ * @return 0 on success, error otherwise
```

Comment:
```suggestion
 * @brief Step through different test tones.
 *
 * @note A stream must already be running to use this feature.
 * Will step through test tones: 1 kHz, 2 kHz, 4 kHz and off.
 *
 * @return 0 on success, error otherwise.
```
Seems tab is not used here, would be nice

Reply:
Has not been fixed

Reply:
Strange. Fixed